src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -176,6 +177,15 @@ public void autonomousInit() {
         }
     }
 
+    @Override
+    public void teleopPeriodic() {
+        if (SubsystemConstants.DRIVE_ENABLED) {
+            subsystems.drivebaseSubsystem.setDefaultCommand(new DriveCommand(subsystems.drivebaseSubsystem,
+                    controls.driveController.getLeftXAxis(), controls.driveController.getLeftYAxis(),
+                    controls.driveController.getRightXAxis()));
+        }
```

Comment:
Please move back to `bindDriveControls()`, the default command should not be set periodically.